### ISSUE TITLE
Create app registration for AWS IAM Identity Centre

### ISSUE THREAD

**Issue by NickCorcoran** [bcgov/cloud-pathfinder#2961]
**Describe the Issue**
In switch from using KeyCloak to direct Azure authentication, we need to create an Azure Enterprise App registration for this purpose.

**Additional Context**
https://docs.aws.amazon.com/singlesignon/latest/userguide/idp-microsoft-entra.html
https://learn.microsoft.com/en-us/entra/identity/saas-apps/aws-single-sign-on-tutorial
https://learn.microsoft.com/en-us/entra/identity/saas-apps/aws-single-sign-on-provisioning-tutorial

**Acceptance Criteria**
- [x] Discover app registration required
- [x] Submit Service Now request to ADMS to create - Nick to be setup as Owner

To be done in separate ticket
- [ ] Setup basic config to be ready to connect to ALZ
- [ ] Try out in current Forge

**Comment by NickCorcoran** [comment]
Waiting on Service now ticket.

**Comment by ThibaultBC** [comment]
@NickCorcoran Any news on this, did you hear from Service Now?  Trying to understand if this ticket is still blocked or just lower priority?

**Comment by NickCorcoran** [comment]
Update from ADMS is that ticket is under technical review and should be completed within a week.

**Comment by NickCorcoran** [comment]
Closing this ticket as IAM Identity Centre is in place.  Will create new ticket to cover testing.
